tests/shutdown/cleanup_before_shutdown.pm

```diff
@@ -59,7 +59,7 @@ END_SCRIPT
     # Proceed with dhcp cleanup on qemu backend only.
     # Cleanup is made, because if same hdd image used in multimachine scenario
     # on several nodes, the dhcp clients use same id and cause conflicts on dhcpd server.
-    if (check_var('BACKEND', 'qemu')) {
+    if (check_var('BACKEND', 'qemu') && !check_var('EXTRATEST','wicked')) {
```

Comment:
Can you explain, why we need this "wicked" exception here?

Reply:
so it is fine for you to delete all xmls in `/var/lib/wicked` ? I prefer to have `wicked` package as it is after installation without any modifications

Reply:
But when I understand that comment correct, then these files are already modified, otherwise it wouldn't make sense to remove them.

Reply:
yes , you right , I double check that with Marius too